MastodonSDK/Sources/MastodonUI/Vendor/MetaTextView+PasteExtensions.swift

```diff
@@ -15,19 +15,17 @@ extension MetaTextView {
         
         // fix #660
         // https://github.com/mastodon/mastodon-ios/issues/660
-        if #available(iOS 15.0, *) {
-            var nextResponder = self.next;
-            
-            // Force the event to bubble through ALL responders
-            // This is a workaround as somewhere down the chain the paste event gets eaten
-            while (nextResponder != nil) {
-                if let nextResponder = nextResponder {
-                    if (nextResponder.responds(to: #selector(UIResponderStandardEditActions.paste(_:)))) {
-                        nextResponder.perform(#selector(UIResponderStandardEditActions.paste(_:)), with: sender)
-                    }
+        var nextResponder = self.next;
```

Comment:
This code could be rewritten using [`sequence(first:next:)`](https://developer.apple.com/documentation/swift/sequence(first:next:)) (although I’m not sure if it needs to be)

```swift
for responder in sequence(first: self, next: \.next) {
    if responder.responds(to: #selector(UIResponderStandardEditActions.paste(_:))) {
        responder.perform(#selector(UIResponderStandardEditActions.paste(_:)), with: sender)
    }
}
```